### DISCUSSION TITLE
Icon request: icon-skateboard

### DISCUSSION THREAD

**Discussion by romanglom** [FortAwesome/Font-Awesome#21148] [Icon Requests]
It would be awesome if you can make a new icon of a skateboard.
The section for this new icon of a skateboard should be in "Transportation Icons", because It's a common icon like the Bicycle.
Thank you very much!

**Comment by TheChen5** [comment]
There is already a request for icon-skate.
Please change it to `icon-skateboard`

**Comment by webhacking** [comment]
+2

**Comment by fwguerra** [comment]
+3

**Comment by gsilvatici** [comment]
+4

**Comment by ldeseta** [comment]
+5

**Comment by abfskateshop** [comment]
+6

**Comment by gni** [comment]
+7

**Comment by MuffinLightning** [comment]
+8

**Comment by NathanGell** [comment]
+9
With electric skateboards becoming more and more popular, this icon will be in high demand.

**Comment by jamesrhenry** [comment]
Yes, I agree, we need a skateboard icon!

**Comment by fraxflax** [comment]
+11

**Comment by fraxflax** [comment]
This should, in my opinion, be an empty skateboard as I also requested a skateboard-rider icon.
Inspiration:
![image](https://user-images.githubusercontent.com/16400874/70711312-856db400-1ce1-11ea-8559-b54484c1af29.png)

**Comment by NPCtom** [comment]
+12

**Comment by alprnyldz** [comment]
+13

**Comment by loiccc** [comment]
+14 and scooters too

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🛹 [U+1F6F9](https://emojipedia.org/emoji/%F0%9F%9B%B9/) Skateboard

**Comment by IAmDanielPeter** [comment]
Sad we're still waiting on this. Absolutely disrespectful. This should've been done already!!

**Comment by krkxone** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by povodniktim** [comment]
+16
